packages/salesforcedx-utils-vscode/test/unit/telemetry/cliConfiguration.test.ts

```diff
@@ -0,0 +1,57 @@
+import { GlobalCliEnvironment } from '../../../src/cli';
```

Comment:
Should we add a copyright statement to the top of this file?

Reply:
👍